packages/mdl-theme/README.md

```diff
@@ -0,0 +1,106 @@
+# mdl-theme
```

Comment:
Looks really great! I only have two suggestions:

- Perhaps it would be useful to show the output CSS of the Sass function with a light explanation of what's going on?
- For dark theming, it may be useful to demonstrate how users can activate dark  mode by using the proper CSS modifier classes on either the elements themselves or an ancestor thereof.

Other than that, looks awesome :D